vcr/api/v2/api.go

```diff
@@ -160,14 +161,19 @@ func (w *Wrapper) SearchIssuedVCs(ctx echo.Context, params SearchIssuedVCsParams
 	}
 
 	foundVCs, err := w.VCR.Issuer().SearchCredential(ssi.URI{}, *credentialType, *issuerDID, subjectID)
-	result := make([]SearchVCResult, len(foundVCs))
-	// TODO: add revocation status
-	for i, resolvedVC := range foundVCs {
-		result[i] = SearchVCResult{VerifiableCredential: resolvedVC}
-	}
 	if err != nil {
 		return err
 	}
+	result := make([]SearchVCResult, len(foundVCs))
+	for i, resolvedVC := range foundVCs {
+		var revocation *Revocation
+		revocation, err = w.VCR.Verifier().GetRevocation(*resolvedVC.ID)
+		if err != nil && !errors.Is(err, verifier.ErrNotFound) {
```

Comment:
if a VC is revoked twice, GetRevocation returns an error, causing SearchResults to return an error (and no way to execute it without it erroring out). Is that OK? Should GetRevocation return a list, or the most recent (active) revocation instead in case there's multiple?

Reply:
I don't like double revocations. I would say only the first one should be registred.

Reply:
Although you can't revoke an already revoked VC (409 on API). This could still happen theoretically. It would be fine to return the first one.

Reply:
This is actually interesting (out of scope for this PR); what if you revoke it with an incorrect revocation date (say, 5 years in future). Does that make the revocation effective after 5 years or immediate? Rationale: if you make a mistake during a revocation that causes the revocation not to be effective, can you issue another revocation that **is** effective?

Reply:
@woutslakhorst ?